server/eclipse-project/src/main/java/us/freeandfair/corla/Main.java

```diff
@@ -247,6 +254,47 @@ private void activateEndpoints() {
   }
   
   /**
+   * Initializes the ASM. If it cannot be read from the database, it is created from 
+   * scratch and committed to the database.
+   * 
+   * @exception IllegalStateException if we can't get the ASM. 
+   */
+  private void initializeASM() throws IllegalStateException {
+    Persistence.beginTransaction();
+    ASM = new RLAToolASM();
+    // note that this resets the ASM every launch right now
+    ASM.stepTransition(RLATransitionFunction.ONE.value()); 
+    
+// the following is some old database code from the initial persistence 
```

Comment:
We can; I can store it somewhere else. I just wanted to remember what I had done for when I have to do it again.